src/sentry/integrations/slack/endpoints/action.py

```diff
@@ -183,17 +184,27 @@ def post(self, request: Request) -> Response:
 
         data = slack_request.data
 
+        # Actions list may be empty when receiving a dialog response
+        action_list = data.get("actions", [])
+
         # if a user is just clicking our auto response in the messages tab we just return a 200
-        if (
-            data.get("actions")
-            and data["actions"][0].get("value", "") == "sentry_docs_link_clicked"
-        ):
+        if action_list and action_list[0].get("value", "") == "sentry_docs_link_clicked":
             return self.respond()
 
         channel_id = slack_request.channel_id
         user_id = slack_request.user_id
         response_url = data.get("response_url")
 
+        if action_list and action_list[0].get("value", "") in ["link", "ignore"]:
+            payload = {"delete_original": "true"}
+            try:
+                post(response_url, json=payload)
```

Comment:
We don't use `requests.post` anywhere else in the repo. Would it make sense to use
```python
slack_client = SlackClient()
payload = {"delete_original": "true"}
try:
    slack_client.post(response_url, data=payload, json=True)
except ApiError as e:
    logger.error("slack.action.response-error", extra={"error": str(e)})
```

Reply:
So I'm not able to use the SlackClient here because the response url is in the following format: `https://hooks.slack.com/actions/<channel_id>/foo/bar`. It a direct url that I need to make a post request to, we have this pattern in our chartcuterie [requests](https://github.com/getsentry/sentry/blob/master/src/sentry/charts/chartcuterie.py#L68). the SlackClient attaches a base url to this and results in a 404 client not found error